app/src/main/java/org/mozilla/fenix/home/pocket/PocketStoriesViewHolder.kt

```diff
@@ -78,20 +75,22 @@ class PocketStoriesViewHolder(
             }
         }
 
-        Column(modifier = Modifier.padding(top = 72.dp)) {
-            SectionHeader(
-                text = stringResource(R.string.pocket_stories_header_1),
-                modifier = Modifier
-                    .fillMaxWidth()
-                    .padding(horizontal = horizontalPadding)
-                    .wrapContentHeight(align = Alignment.Top)
+        Column(
+            modifier = Modifier.padding(
+                top = 72.dp,
+                start = horizontalPadding,
+                end = horizontalPadding
```

Comment:
Is this expected this would add extra horizontal padding to `PocketStories`?

Reply:
Yes, it was previously added in each composable separately, but that needed an extra modifier param in `HomeSectionHeader` just for this one case.